src/components/App.js

```diff
@@ -29,6 +29,7 @@ const App = () => {
                 <Route path='/task03' component={Task03}/>
                 <Route path='/task04' component={Task04}/>
                 <Route path='/task05' component={Task05}/>
+                <Route path='/404.html' component={NotFound}/>
```

Comment:
Jeśli chcesz, aby "wygląd" strony produktu wyglądał inaczej to wystarczy w tym miejscu zdefiniować osobną ścieżkę, a w w miejscu gdzie renderujesz produkty, przekażesz ścieżkę np.

```
<Route path='/product/:id' component={NotFound}/>
```

```
<Product version="full" path="/product" ... />
```

Reply:
chyba nie za bardzo rozumiem, czemu mam cos w linkach ruszac?

Reply:
Zrozumiałem Twoje pytanie w ten sposób, że chcesz mieć kompletnie "czystą" stronę z danami produktu - wtedy musiałbyś faktycznie zmienić linki. Natomiast tak jak to zrobiłeś jest to wystarczające - nie trzeba nic kombinować z linkami :)